doc/tutorial/introduction/index.md

```diff
@@ -151,7 +152,12 @@ algorithm which then executes all alignments in parallel and stores the results
 objects. The second argument to seqan3::align_pairwise is the *configuration* which allows you to specify
 a lot of parameters for the alignment computation, for instance score functions, banded alignment and whether
 you wish to compute a traceback or not. The configurations have their own namespace seqan3::align_cfg and can
-be combined via the logical OR operator (`|`) for building combinations. Check out the alignment tutorial if you want to learn more.
+be combined via the logical OR operator (`|`) for building combinations. Check out the alignment tutorial if you want
+to learn more.
+
+\note
+In general, we omit `using namespace seqan3;`, to show which parts are from SeqAn3 and which aren't. However, for comprehensibility we declare string literals like _dna5 beforehand with a using directive. You can skip it, if you
+work with the overall namespace or declare it in lines 20 and 21 directly.
```

Comment:
We encourage you to **avoid** declaring `using namespace seqan3;`. This has the additional benefit of easily distinguishing between library features and standard C++. The only exception are string literals, where we often use `using seqan3::operator""_dna4` for convenience.